ChatQnA/kubernetes/intel/README.md

```diff
@@ -9,13 +9,20 @@
 > You need to make sure you have created the directory `/mnt/opea-models` to save the cached model on the node where the ChatQnA workload is running. Otherwise, you need to modify the `chatqna.yaml` file to change the `model-volume` to a directory that exists on the node.
 >
 > File upload size limit: The maximum size for uploaded files is 10GB.
+> 
+> The `/mnt/opea-models` directory is used to cache the model files to avoid downloading the model files every time the pod is restarted.
+> 
+> The directory should have the following permission:
+> 
+> `sudo chown 1000:1000 /mnt/opea-models`
```

Comment:
I think this should 1000:0 based on the securityContext settings.

Reply:
I will refactor the solution to use local-path-provisioner and this requirement will go away.